pkg/runtime/cli.go

```diff
@@ -148,3 +150,17 @@ func FromFlags() (*DaprRuntime, error) {
 	}
 	return NewDaprRuntime(runtimeConfig, globalConfig), nil
 }
+
+// getDefaultComponentsFolder returns the hidden .components folder created at init time
+func getDefaultComponentsFolder() string {
+	const daprDirName = ".dapr"
+	const daprWindowsOS = "windows"
+	const componentsDirName = "components"
+	daprDirPath := os.Getenv("HOME")
+	if runtime.GOOS == daprWindowsOS {
+		daprDirPath = os.Getenv("USERPROFILE")
+	}
+
+	defaultComponentsPath := filepath.Join(daprDirPath, daprDirName, componentsDirName)
```

Comment:
nit: we do not need to assign new variable.

Reply:
fixed